x-pack/legacy/plugins/code/public/components/main/clone_status.tsx

```diff
@@ -17,18 +20,29 @@ interface Props {
 
 export const CloneStatus = (props: Props) => {
   const { progress: progressRate, cloneProgress, repoName } = props;
-  let progress = `Receiving objects: ${progressRate.toFixed(2)}%`;
+  let progress = `${i18n.translate(
+    'xpack.code.mainPage.content.cloneStatus.progress.receivingText',
+    { defaultMessage: 'Receiving objects' }
+  )}: ${progressRate.toFixed(2)}%`;
```

Comment:
It would be better to insert the `progressRate` as a value in the translation itself. (Same for the translations following this pattern)

```
{
  defaultMessage: `Receiving objects: {progressRate}%`,
  values: { progressRate:  progressRate.toFixed(2) },
} 
```

Reply:
done